src/pillowtalk_impl.c

```diff
@@ -1040,3 +1080,5 @@ static void generate_node_json(pt_node_t* node, yajl_gen g)
     yajl_gen_null(g);
   }
 }
+
+/* vim: set tabstop=2 shiftwidth=2: */
```

Comment:
Agree with `.editorconfig`.

And... to be honest, i only have few capability on c programming. I am happy to hear and learn anything about c programming!

Is there any guideline for the coding style?